examples/docs-examples/examples/executing-tasks/foreach.mjs

```diff
@@ -2,7 +2,7 @@ import { TaskExecutor, pinoPrettyLogger } from "@golem-sdk/task-executor";
 
 (async () => {
   const executor = await TaskExecutor.create({
-    package: "529f7fdaf1cf46ce3126eb6bbcd3b213c314fe8fe884914f5d1106d4",
+    package: "golem/node:20-alpine",
```

Comment:
if the task does not include any nodejs command,I think it is better to use the image `golem/alpine:latest`
```suggestion
    package: "golem/alpine:latest",
```